vividus-plugin-web-app-to-rest-api/src/test/java/org/vividus/bdd/steps/integration/ResourceCheckStepsTests.java

```diff
@@ -193,7 +196,11 @@ void shouldCheckResourcesFromPages() throws IOException, InterruptedException, E
         resourceCheckSteps.init();
         ExamplesTable examplesTable =
                 new ExamplesTable("|pages|\n|https://first.page|\n|https://second.page|");
-        when(webApplicationConfiguration.getMainApplicationPageUrl()).thenReturn(VIVIDUS_URI);
+        lenient().doAnswer(call ->
+        {
+            Sleeper.sleep(5, TimeUnit.SECONDS);
```

Comment:
isn't too much?

Reply:
the exact min value is hard to determine, I would need to run the test ~100 times to ensure that the test is not flaky with given sleep time, to avoid spending my time for this I decided to take 5 sec since it's 100% enough

Reply:
Have you noticed it failing with 1 sec?

Reply:
I preferred to save my time, feel free to try it out and if it will succeed in 100/100 I'll change it to 1 sec

Reply:
But it will make our CI slower